driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -43,11 +43,11 @@ impl LinearOptimisationPriceFinder {
     }
 }
 
-fn token_id(token: u8) -> String {
+pub fn token_id(token: u8) -> String {
```

Comment:
Why is this necessary? I don't see this method being called outside of this model.